handsontable/src/3rdparty/walkontable/src/utils/stylesHandler.js

```diff
@@ -118,6 +119,18 @@ export class StylesHandler {
    * @returns {number} The calculated row height.
    */
   getDefaultRowHeight() {
+    const cssVarRowHeightValue = this.getCSSVariableValue('row-height');
+
+    if (!this.#isClassicTheme && !cssVarRowHeightValue) {
+      warn(`The "${this.#themeName}" theme is enabled, but its stylesheets are missing. \
+Import the correct CSS files in order to use that theme.`);
+
+      this.#isClassicTheme = true;
+      this.useTheme();
+
+      return CLASSIC_THEME_DEFAULT_HEIGHT;
+    }
+
     if (this.#isClassicTheme) {
       return CLASSIC_THEME_DEFAULT_HEIGHT;
     }
```

Comment:
Wouldn't it be more efficient if you returned a default height of the row for the classic theme ASAP and then processed the rest? 
```suggestion
    if (this.#isClassicTheme) {
      return CLASSIC_THEME_DEFAULT_HEIGHT;
    }
    
    const cssVarRowHeightValue = this.getCSSVariableValue('row-height');

    if (!this.#isClassicTheme && !cssVarRowHeightValue) {
      warn(`The "${this.#themeName}" theme is enabled, but its stylesheets are missing. \
Import the correct CSS files in order to use that theme.`);

      this.#isClassicTheme = true;
      this.useTheme();

      return CLASSIC_THEME_DEFAULT_HEIGHT;
    }
```